### PR TITLE
14.0 mig pos backend partner rebased

### PR DESCRIPTION
**Pull request by sebastienbeau**

### CODE REVIEW

**Review by sebastienbeau — COMMENTED** [review]
PR is ready and work please review
LGTM approved

**Review by PierrickBrun — APPROVED** [review]

**Comment by PierrickBrun** [comment]
@sebastienbeau I had a problem with the order of inheritance so I splited the tools.js file. (The problem did not appear in debug=assets mode)

**Comment by sebastienbeau** [comment]
@hparfr can you review so we can merge it ?

File: pos_backend_partner/static/src/js/pos_partner.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+*/
+
+odoo.define("pos_backend_partner.ProductScreen", function (require) {
+    "use strict";
+    var Registries = require("point_of_sale.Registries");
+    var ProductScreen = require("point_of_sale.ProductScreen");
+    var PaymentScreen = require("point_of_sale.PaymentScreen");
+    var tools = require("pos_backend_communication.tools");
+
+    var pos_instance = undefined;
```
Comment (sebastienbeau) [inline pos_backend_partner/static/src/js/pos_partner.js:14]: @hpar pos_instance and _t seem to raise en issue for pylint https://github.com/OCA/pos/runs/6715157129?check_suite_focus=true#step:7:85

useless error ?

Comment (hparfr) [reply pos_backend_partner/static/src/js/pos_partner.js:14]: it seams so, they are used l40

**Pull request closed without merging**